CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 Note: Minor version `0.X.0` update might break the API, It's recommended to pin geojson-pydantic to minor version: `geojson-pydantic>=0.6,<0.7`
 
+## [1.1.2] - 2024-10-22
```

Comment:
to me this PR brings a `patch` and fix a wrong behaviour which is why I chose to make a new `patch` version

Happy to make this a new minor version 🤷

Reply:
I'm 👍🏼 on patch, I would be very surprised if anyone was relying on the (wrong) validation error.